tests/Database/Traits/ValidationTest.php

```diff
@@ -20,13 +23,16 @@ protected function getConnectionName()
         return 'mysql';
     }
 
-    protected function getTable()
+    protected function getTable($table = null)
     {
         return 'users';
     }
 
     public function testUnique()
     {
+        // Fake a request so flash messages are not sent
+        Input::swap(new Request());
```

Comment:
Isn't the Input facade just a wrapper around the Request facade? We should avoid using it at all, it exists purely as a BC layer if I recall correctly

Reply:
It is being used in the validation traits that these tests (and the ones below) are for, so we'd have to change them there.